### PR TITLE
feature(component): 218 Events card

### PR DESCRIPTION
**Pull request by codeuri**
- Generic card refactoring
- Created Card Events component based on Generic card
- Generic card tests fix/update
- Card Events component tests
- Added MD for the new component
- Styles

### CODE REVIEW

**Comment by gepifanio** [comment]
Comment: Deploy Started

Building with commit 3b1931608efedf0f189616d9687b6db50f6b56fa

https://app.netlify.com/sites/pattern-lib-unimelb/deploys/5c9fda8f140f130008916221

File: components/cards/_card-events.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      width: 18px;
+      height: 18px;
+    }
+
+    @media (--bp-max-mobile) {
+      border-bottom: 1px solid;
+    }
+  }
+
+  .card__excerpt {
```
Comment (TimMansell) [inline components/cards/_card-events.css]: Missing #575756 text colour.

File: components/cards/_card-events.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
+.card-events {
+  color: rgb(var(--col-brand));
+
+  .card__title {
+    font-size: 20px;
+    font-weight: var(--fw-semibold);
```
Comment (TimMansell) [inline components/cards/_card-events.css]: should be 700 font-weight.  We don't have a var for that so will need to create one.

File: components/cards/_card-events.css [hunk 3]
```diff
@@ -0,0 +1,30 @@
+.card-events {
+  color: rgb(var(--col-brand));
+
+  .card__title {
```
Comment (TimMansell) [inline components/cards/_card-events.css]: Should be #000 text colour

File: components/cards/_card-events.css [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    font-size: 20px;
+    font-weight: var(--fw-semibold);
+    text-decoration: none;
+  }
+
+  .card__sub-titles {
```
Comment (TimMansell) [inline components/cards/_card-events.css]: Should be text colour #575756

File: components/cards/_card-events.css [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    margin-bottom: .25rem;
+    border-top: 1px solid;
```
Comment (TimMansell) [inline components/cards/_card-events.css]: border colour should be #C6C8CA

File: components/cards/_card-events.css [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+    padding: .5rem 0;
+
+    svg {
+      width: 18px;
+      height: 18px;
+    }
+
+    @media (--bp-max-mobile) {
+      border-bottom: 1px solid;
```
Comment (TimMansell) [inline components/cards/_card-events.css]: border colour should be #C6C8CA

File: components/cards/stories/Events/Story1.vue [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+            <SvgIcon name="calendar"/>
+            <span>March 31 1992</span>
+          </div>
+          <div
+            slot="sub-title-3"
+            class="sub-title">
+            <span>Theme: Lorem ipsum.</span>
+          </div>
+          <template slot="links">
+            <a href="/">View generic details ></a>
```
Comment (TimMansell) [inline components/cards/stories/Events/Story1.vue]: Should use `<SvgIcon>` chevron

File: components/cards/CardEvents.vue [hunk 8]
```diff
@@ -0,0 +1,45 @@
+<template>
+  <GenericCard
+    :thumb="thumb"
+    :title="title"
+    :href="href"
+    :excerpt="excerpt"
+    :cols="1"
+    :class-name="'card-events ' + className"
```
Comment (TimMansell) [inline components/cards/CardEvents.vue]: use card--events modifier instead

**Comment by jaredh-um** [comment]
@TimMansell 

Card issues
Generic cards:
Border-radius on card background missing
Box-shadow missing

Event cards:
Divider line should be grey as per design for all variations
Font colours are default for UoM and standard overrides for FFAM as you've mentioned above

File: components/cards/_card-events.css (hunk 3 above)
Comment (gepifanio) [reply components/cards/_card-events.css]: In this case, the text colour should be added here or the override on `ffam-prototype` repo?

Comment (TimMansell) [reply components/cards/_card-events.css]: ffam-prototype

File: components/cards/_card-events.css (hunk 4 above)
Comment (TimMansell) [reply components/cards/_card-events.css]: change in ffam-prototype.

File: components/cards/_card-events.css (hunk 5 above)
Comment (TimMansell) [reply components/cards/_card-events.css]: change in ffam-prototype.

File: components/cards/_card-events.css (hunk 6 above)
Comment (TimMansell) [reply components/cards/_card-events.css]: change in ffam-prototype.

File: components/cards/_card-events.css (hunk 1 above)
Comment (TimMansell) [reply components/cards/_card-events.css]: change in ffam-prototype.

**Pull request merged**